addons/bus/static/src/services/bus_service.js

```diff
@@ -73,6 +73,8 @@ export const busService = {
                 isInitialized = true;
                 connectionInitializedDeferred.resolve();
                 return;
+            } else if (type === "reconnect") {
+                notificationBus.trigger(type, {});
```

Comment:
The bus service already provides the `reconnect` event via `busService.addEventListener("reconnect", callback)`. `notificationBus` should only trigger notifications (i.e. messages coming through the bus).